### PR TITLE
OSDOCS#4551: Add the common terms for networking book

### PR DESCRIPTION
**Pull request by xenolinux**
Version(s): 4.8+

Issue: https://issues.redhat.com/browse/OSDOCS-4551

Link to docs preview: https://52765--docspreview.netlify.app/openshift-enterprise/latest/networking/understanding-networking.html#nw-networking-glossary-terms_understanding-networking

QE review:
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

File: modules/nw-networking-glossary-terms.adoc [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+
+pod::
+One or more containers with shared resources, such as volume and IP addresses, running in your {product-title} cluster.
+A pod is the smallest compute unit defined, deployed, and managed.
+
+PTP Operator::
+The PTP Operator creates and manages the linuxptp services.
+
+replicas::
+`replicas` is the desired number of Ingress Controller replicas.
```
[the remark this answers is not in the document]
Comment (xenolinux) [reply modules/nw-networking-glossary-terms.adoc]: Addressed

**Comment by xenolinux** [comment]
@lihongan I addressed the comment, does this look good now?

**Comment by lihongan** [comment]
@xenolinux ingress/router part looks good. 
For other networking part, we need @anuragthehatter @zhaozhanqi to help take a look. thanks.

**Pull request closed without merging**